packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter.ts

```diff
@@ -590,7 +595,7 @@ export class StripeProductsAndPlansConverter {
             await this.writeToFileProductConfig(
```

Comment:
At the moment the script creates 2 `.json` files for Product config. That's probably cause we call `this.writeToFileProductConfig` twice in the script, and only 1 instance has the new name.

Reply:
Ah, got the other one.